.lintrunner.toml

```diff
@@ -2563,6 +2563,7 @@ exclude_patterns = [
     'torch/utils/viz/__init__.py',
     'torch/utils/viz/_cycles.py',
     'torch/utils/weak.py',
+    'torch/xpu/_gpu_trace.py',
```

Comment:
Why is this skipped?

Reply:
Similar to CUDA, some codes in `_gpu_trace.py` will be formatted into one line or two lines. If not skipped, it would affect the cleanliness of the code.